src/CaptureClient/ApiEventProcessor.cpp

```diff
@@ -27,132 +27,192 @@ inline std::string DecodeString(const Source& encoded_source) {
                                  encoded_source.encoded_name_additional_size());
 }
 
-inline TimerInfo TimerInfoFromEncodedEvent(const orbit_api::EncodedEvent& encoded_event,
-                                           uint64_t start, uint64_t end, int32_t pid, int32_t tid,
-                                           uint32_t depth) {
-  TimerInfo timer_info;
-  timer_info.set_start(start);
-  timer_info.set_end(end);
-  timer_info.set_process_id(pid);
-  timer_info.set_thread_id(tid);
-  timer_info.set_depth(depth);
-  timer_info.set_type(TimerInfo::kApiEvent);
-  timer_info.mutable_registers()->Reserve(6);
-  timer_info.add_registers(encoded_event.args[0]);
-  timer_info.add_registers(encoded_event.args[1]);
-  timer_info.add_registers(encoded_event.args[2]);
-  timer_info.add_registers(encoded_event.args[3]);
-  timer_info.add_registers(encoded_event.args[4]);
-  timer_info.add_registers(encoded_event.args[5]);
-  return timer_info;
+void EncodedColorToColor(uint32_t encoded_color, orbit_client_protos::Color* color) {
+  color->set_red((encoded_color & (0xff << 24)) >> 24);
+  color->set_green((encoded_color & (0xff << 16)) >> 16);
+  color->set_blue((encoded_color & (0xff << 8)) >> 8);
+  color->set_alpha(encoded_color & 0xff);
 }
 }  // namespace
 
 ApiEventProcessor::ApiEventProcessor(CaptureListener* listener) : capture_listener_(listener) {
   CHECK(listener != nullptr);
 }
 
-void ApiEventProcessor::ProcessApiEvent(const orbit_grpc_protos::ApiEvent& grpc_api_event) {
+void ApiEventProcessor::ProcessApiEventLegacy(const orbit_grpc_protos::ApiEvent& event_buffer) {
```

Comment:
event_buffer makes the reader think there are multiple events contained in a buffer, why the name change?